lib/components/tabs/tabview.coffee

```diff
@@ -257,10 +257,12 @@ module.exports = class KDTabView extends KDScrollView
   # DEFAULT ACTIONS
   handleClicked: (event, handle) ->
 
-    {pane} = handle.getOptions()
+    { pane }  = handle.getOptions()
+    target    = $ event.target
 
     # fixme: make close icon a kdview and check the view instead.
```

Comment:
instead of adding another nested element, let's do it as I commented here over a year ago :)